### PR TITLE
Add base partial charge setting things

### PR DESCRIPTION
**Pull request by IAlibay**
## Developers certificate of origin
- [x] I certify that this contribution is covered by the MIT License [here](https://github.com/OpenFreeEnergy/openfe/blob/main/LICENSE) and the **Developer Certificate of Origin** at <https://developercertificate.org/>.

### CODE REVIEW

File: openfe/tests/protocols/test_openmm_equil_rfe_protocols.py [hunk 1]
```diff
@@ -1407,7 +1407,7 @@ def test_get_estimate(self, protocolresult):
         est = protocolresult.get_estimate()
 
         assert est
-        assert est.m == pytest.approx(16.85, abs=0.3)
+        assert est.m == pytest.approx(15.71, abs=0.3)
```
Comment (IAlibay) [inline openfe/tests/protocols/test_openmm_equil_rfe_protocols.py:1410]: @hannahbaumann could you double check what value you get here when re-running the generator for this? I've always gotten ~ 16.8 for this sytem, it's really odd that things changed by a whole kcal/mol 🤯

Comment (IAlibay) [reply openfe/tests/protocols/test_openmm_equil_rfe_protocols.py:1410]: Just to add here - I ran this multiple times and consistently got ~ 15.7

Comment (IAlibay) [reply openfe/tests/protocols/test_openmm_equil_rfe_protocols.py:1410]: From today's call - we didn't guard against elf10 in partial charge generation...

Comment (IAlibay) [reply openfe/tests/protocols/test_openmm_equil_rfe_protocols.py:1410]: This definitely was down to oechem vs ambertools...

**Review by richardjgowers — APPROVED** [review]

**Pull request merged**